x-pack/elastic-agent/pkg/agent/cmd/enroll.go

```diff
@@ -59,6 +60,7 @@ func addEnrollFlags(cmd *cobra.Command) {
 	cmd.Flags().Uint16P("fleet-server-port", "", 0, "Fleet Server HTTP binding port (overrides the policy)")
 	cmd.Flags().StringP("fleet-server-cert", "", "", "Certificate to use for exposed Fleet Server HTTPS endpoint")
 	cmd.Flags().StringP("fleet-server-cert-key", "", "", "Private key to use for exposed Fleet Server HTTPS endpoint")
+	cmd.Flags().StringSliceP("header", "", []string{}, "App auth token used for elasticsearch")
```

Comment:
Documentation for this flag is incorrect, seems like a copy/paste error.

Reply:
thanks for noticing